invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -19,8 +19,8 @@ export const RDMRecordResultsListItem = ({ result, index }) => {
   );
   const publicationDate = _get(
     result,
-    "metadata.publication_date",
-    "No metadata"
+    "ui.publication_date_l10n.value",
```

Comment:
Question: Shouldn't this just be ``ui.publication_date_l10n``?

Reply:
Yes I just kept the dictionary structure but I can change it yes :) But, wouldn't it better just to take from the `metadata.publication_date_l10n` instead of serializing in the `ui` object?